src/main/kotlin/org/digma/intellij/plugin/ui/jcef/JCefComponent.kt

```diff
@@ -77,7 +84,18 @@ private constructor(
             }
         }
 
+        authInfoChangeListener = object : AuthInfoChangeListener {
+            override fun authInfoChanged(authInfo: AuthInfo) {
+                try {
+                    sendUserInfo(jbCefBrowser.cefBrowser, authInfo.userId)
+                } catch (e: Throwable) {
+                    ErrorReporter.getInstance().reportError("JCefComponent.userChanged", e)
+                }
+            }
+        }
+
```

Comment:
see comment about fun interface , you can use like that. and no need to keep the instance as member
```
AuthManager.getInstance().addAuthInfoChangeListener({ authInfo ->
            try {
                sendUserInfo(jbCefBrowser.cefBrowser, authInfo.userId)
            } catch (e: Throwable) {
                ErrorReporter.getInstance().reportError("JCefComponent.userChanged", e)
            }
        },authInfoChangeListenerDisposable)
```